### PR TITLE
Platform_APIs_ERROR_500

### PR DESCRIPTION
**Pull request by kkarthik4cs**
Issue #SB-2868 fix: review comment

### CODE REVIEW

File: common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java [hunk 1]
```diff
@@ -1010,4 +1010,18 @@ public static Map convertJsonStringToMap(String jsonString) throws IOException {
     map.put(ENDDATE, new SimpleDateFormat(YYYY_MM_DD_FORMATTER).format(cal.getTime()));
     return map;
   }
+  
+  /**
+  * This method util will be used to create ProjectCommonException for all kind
```
Comment (manzarul) [inline common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java]: This method util  to This method

File: common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java [hunk 2]
```diff
@@ -1010,4 +1010,18 @@ public static Map convertJsonStringToMap(String jsonString) throws IOException {
...
+  * of client error for the given response code(enum).
+  *
+  * @param ResponseCode
```
Comment (manzarul) [inline common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java]: define ResposneCode :  An enum of all the api responses.

File: common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java [hunk 3]
```diff
@@ -1010,4 +1010,18 @@ public static Map convertJsonStringToMap(String jsonString) throws IOException {
...
+  *            .
```
Comment (manzarul) [inline common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java]: remove  "."

File: common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java [hunk 4]
```diff
@@ -1010,4 +1010,18 @@ public static Map convertJsonStringToMap(String jsonString) throws IOException {
...
+  * @return ProjectCommonException .
+  */
+  public static ProjectCommonException createClientException(ResponseCode exception) {
+	throw new ProjectCommonException(exception.getErrorCode(), exception.getErrorMessage(),
```
Comment (manzarul) [inline common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java]: change throw to return,
exception to responseCode

Comment (kkarthik4cs) [reply common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java]: done

File: common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java (hunk 3 above)
Comment (kkarthik4cs) [reply common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java]: done

File: common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java (hunk 2 above)
Comment (kkarthik4cs) [reply common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java]: done

File: common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java (hunk 1 above)
Comment (kkarthik4cs) [reply common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java]: done

**Pull request merged**